praw/reddit.py

```diff
@@ -367,11 +367,13 @@ def get(self, path, params=None):
         data = self.request('GET', path, params=params)
         return self._objector.objectify(data)
 
-    def info(self, fullnames):
+    def info(self, fullnames=None, url=None):
         """Fetch information about each item in ``fullnames``.
```

Comment:
If it fits can we add "\`\`fullnames\`\`, or from \`\`url\`\`". If it doesn't fit, perhaps we can simply say, "Fetch information from the provided parameters".